README.md

```diff
@@ -1,2 +1,55 @@
-# The-Next-Web
-A clone application of the next web original website
+
+# HTML/CSS capstone project
+
+This project is a responsive website application developed as a completion of HTML & CSS section of Microverse main curriculum. It is a collobtative project intended to showcase HTML & CSS Skills. It is a real-world-like project built with some business specifications. It is a clone of the thenextweb website. It consist of only one page, that is home page. It is adaptive to different standard screen size ( mobile , tablet and desktop). It is developed with HTML and CSS solo.
+
+## Desktop
+
+![screenshot](./img/readme-img1.png)
+
+## Tablet
+
+![screenshot](./img/readme-img2.png)
+
+## Mobile
+
+![screenshot](./img/readme-img3.png)
+
+## Built With
+
+- HTML,
+- CSS
+
+## Live Demo
+
+<a href="https://rawcdn.githack.com/prolajumokeoni/The-Next-Web/9d56f4ded445fa451a1ce60a064939def16ca662/index.html">Live version</a>
+
+## Author
+
+👤 **Oni Olajumoke Priscilla and Tedros Tesfay**
+
+- Github: [Tedros Tesfay](https://github.com/tta2yta)
+- Emaii: tta2yta@gmail.com
```

Comment:
Is it `Emaii`? rather than being `Email` ?

Reply:
We added links to our Linkedin accounts and removed the emails.